x/pki/client/cli/query_noc_certificates.go

```diff
@@ -45,31 +50,66 @@ func CmdListNocIcaCertificates() *cobra.Command {
 	return cmd
 }
 
-func CmdShowNocIcaCertificates() *cobra.Command {
-	var vid int32
+func CmdShowNocCertificates() *cobra.Command {
+	var (
+		vid          int32
+		subject      string
+		subjectKeyID string
+	)
+
 	cmd := &cobra.Command{
-		Use:   "noc-x509-ica-certs",
-		Short: "Gets NOC ICA certificates (ICACs) by VID",
-		Args:  cobra.ExactArgs(0),
+		Use: "noc-x509-cert",
+		Short: "Gets certificates (either root or ica) by one of property combinations: " +
+			"'subject + subject-key-id' or 'VID and subject-key-id' or just 'subject-key-id'",
+		Args: cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			clientCtx := client.GetClientContextFromCmd(cmd)
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			if subject != "" && subjectKeyID != "" {
```

Comment:
What if all three args are passed (vid, subject, SKID)? I think we need to return an error in this case?